### DISCUSSION TITLE
Project Roadmap

### DISCUSSION THREAD

**Discussion by moficodes** [encoredev/encore#12] [Help] [answered]
The project looks great. 

Is there any project roadmap and are you involving the community in the development process?

**Comment by eandre** [comment, marked as answer]
Thanks @moficodes. We definitely want to involve the community in the development process! We're still in the process of setting everything up, and a big part of why we haven't published any roadmap is because we want to source the biggest needs from the community itself. So a roadmap is coming but isn't quite ready yet.

**Reply by marcuskohlberg** [reply]
@moficodes Thanks for getting involved. Is there anything in particular that you're interested in seeing?

**Reply by marcuskohlberg** [reply]
@moficodes There's now a [public roadmap](https://encore.dev/roadmap).
For contributions, there's discussions happening on [Slack](https://encore.dev/slack) and we're going to start creating tickets on here to make it easy to get going with contributing.

**Comment by eandre** [comment]
There is now a [Public Roadmap](https://encore.dev/roadmap) and we've started to tag everything to make it easier to contribute!
